lib/stat_tracker.rb

```diff
@@ -3,12 +3,18 @@
 require_relative 'team'
 require 'pry'
 require_relative 'game_team'
+require_relative 'team_stats'
+require_relative 'game_stats'
+require_relative 'game_team_stats'
 
 class StatTracker
+  include TeamStats, GameStats, GameTeamStats
+
   attr_accessor :teams,
                 :games
 
-  attr_reader :game_teams
+  attr_reader :game_teams,
+              :team_stats
```

Comment:
Just noticed this team_stats attr. What instance variable is this connected to? Or what does this return when called?